### PR TITLE
Get Power BI dataflow last refresh

### PR DESCRIPTION
**Pull request by elic-eon**
### 🤔 Why?

Get last refresh like we did for PBI dataset

### 🤓 What?

Get last success refresh time from the API.

### 🧪 Tested?

Verified the output MCE

### CODE REVIEW

File: metaphor/power_bi/extractor.py [hunk 1]
```diff
@@ -737,6 +745,24 @@ def _find_last_completed_refresh(
 
         return safe_parse_ISO8601(refresh.endTime)
 
+    @staticmethod
+    def _find_refresh_time_from_transaction(
```
Comment (mars-lan) [inline metaphor/power_bi/extractor.py:749]: We seem to have accumulated quite a few of these static util methods. Consider moving them to a separate file to improve readability & testability?

Comment (mars-lan) [reply metaphor/power_bi/extractor.py:749]: Let's do that in a follow-up PR.

**Review by mars-lan — COMMENTED** [review]
Bump up versnion?

**Review by alyiwang — APPROVED** [review]
LGTM

File: metaphor/power_bi/extractor.py (hunk 1 above)
Comment (elic-eon) [reply metaphor/power_bi/extractor.py:793]: SG

**Review by alyiwang — APPROVED** [review]

**Review by mars-lan — APPROVED** [review]

**Pull request merged**